packages/exchange/tests/providers/conftest.py

```diff
@@ -70,6 +73,70 @@ def default_google_env(monkeypatch):
         monkeypatch.setenv("GOOGLE_API_KEY", GOOGLE_API_KEY)
 
 
+# Use a literal string for long or multi-line strings
+class LiteralString(str):
+    pass
+
+
+def literal_presenter(dumper, data):
```

Comment:
Maybe a comment on what this does. IIUC it forces mutliline string

Reply:
thanks for the feedback, I changed the function name and put some comments on it. hope it is better!